### PR TITLE
Update name from Lori to Jamie

### PR DESCRIPTION
**Pull request by tyleramos**

### CODE REVIEW

**Comment by tublitzed** [comment]
Why not just change this from 'Jamie' to 'New Developer' so we don't have to change it each time we hire a new person?

**Comment by tyleramos** [comment]
I considered it but thought it sounded a little cold.  I'm happy to do that though.

File: brainstorming/plan_for_jamie.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ We're going to try to get those in during her first week as well.
 
 ## Weeks 2-4 - Optional Dev Topics
 
-Everything else will be optional, and up to Lori to schedule.  We'll encourage her to have no more than one per day.  Topics offered by developers are the following, but let Mason know if you would like to add anything to the list:
+Everything else will be optional, and up to Jamie to schedule.  We'll encourage her to have no more than one per day.  Topics offered by developers are the following, but let Mason know if you would like to add anything to the list:
```
Comment (fatcatt316) [inline brainstorming/plan_for_jamie.md]: What about just `up to the new hire to schedule` or `up to teh n00b to schedule` ?

**Comment by fatcatt316** [comment]
Also, @tyleramos do you mind changing this part, too? Maybe could just change `her` to `their`:
```
We're going to try to get those in during her first week as well.
```

**Comment by tyleramos** [comment]
Yep, I caught that when removing Jamie's name.  It shouldn't be gender specific now.

File: brainstorming/plan_for_jamie.md [hunk 2]
```diff
@@ -12,11 +12,11 @@ Developers commonly listed three topics that should be required for all new dev
 * Our PR lifecycle/process (Coop)
 * Testing in Wordsmith (Tyler)
 
-We're going to try to get those in during her first week as well.
+We're going to try to get those in during the first week as well.
 
 ## Weeks 2-4 - Optional Dev Topics
 
-Everything else will be optional, and up to Lori to schedule.  We'll encourage her to have no more than one per day.  Topics offered by developers are the following, but let Mason know if you would like to add anything to the list:
+Everything else will be optional, and up to the new developer to schedule.  We'll encourage them to have no more than one per day.  Topics offered by developers are the following, but let Mason know if you would like to add anything to the list:
```
Comment (dneanover) [inline brainstorming/plan_for_jamie.md]: Them creates a singular/plural disagreement.  Perhaps him/her instead?

Comment (tyleramos) [inline brainstorming/plan_for_jamie.md]: you right!

**Pull request merged**